test/unit/ajax.js

```diff
@@ -2840,6 +2840,32 @@ if ( typeof window.ArrayBuffer === "undefined" || typeof new XMLHttpRequest().re
 		} );
 	} );
 
+	QUnit.test( "jQuery.append with crossOrigin attribute", function( assert ) {
```

Comment:
The attribute is lowercase so:
```suggestion
	QUnit.test( "jQuery.append with crossorigin attribute", function( assert ) {
```